rmgpy/rmg/model.py

```diff
@@ -1209,21 +1209,23 @@ def removeSpeciesFromEdge(self, reactionSystems, spec):
         """
         Remove species `spec` from the reaction model edge.
         """
-
         # remove the species
         self.edge.species.remove(spec)
         self.indexSpeciesDict.pop(spec.index)
 
         # clean up species references in reactionSystems
         for reactionSystem in reactionSystems:
-            reactionSystem.speciesIndex.pop(spec)
+            try:
+                reactionSystem.speciesIndex.pop(spec)
+            except:
+                pass
```

Comment:
If spec is None it will fail on the first line of the function.  I'll make it only ignore KeyError's.